core/src/test/java/com/graphhopper/reader/OSMReaderTest.java

```diff
@@ -648,7 +648,7 @@ public long handleNodeTags( OSMNode node )
         OSMWay way = new OSMWay(1L);
         way.getNodes().add(1);
         way.getNodes().add(2);
-        way.setTag("highway", "motorway");
+        way.setTag("route", "ferry");
```

Comment:
I mean: why is this change necessary? Maybe we should add a new test specific for route=ferry? (and avoid removing the test for highway=motorway?)